### ISSUE TITLE
reload data-turbolinks-permanent section sometimes

### ISSUE THREAD

**Issue by BigGitWorld** [turbolinks/turbolinks#590]
Hi.
I'm going to create an E-commerce web application that have a sidebar including different filters for different categories.
I'm using ``data-turbolinks-permanent`` to keep sidebar fragment (div) from changes generally. but I need to change it sometimes (for example when category will be changed).
So, I'm looking for a solution to be able to reload the ``data-turbolinks-permanent`` div fragment manually.
For example, I think it would be nice if the turbolinks could replace ``data-turbolinks-permanent`` section by the new HTML content, if the new one (coming from server) does not have this attribute (``data-turbolinks-permanent``).
What do you think?
Is there anyway to do it now?

**Comment by oscar-al** [comment]
Test

**Comment by rahul-bluebash** [comment]
@BigGitWorld Yes, I am facing the same issue. Any solution or workaround please ?
